ocaml/idl/datamodel_pool.ml

```diff
@@ -1331,6 +1331,9 @@ let t =
             "coordinator_bias"
             "true if bias against pool master when scheduling vms is enabled, \
              false otherwise"
+        ; field ~qualifier:DynamicRO ~lifecycle:[] ~ty:DateTime
```

Comment:
Is this best practice? We have an optional parameter and still use special value as default rather than `None`

Reply:
Hi @lindig , sorry, do you suggest to use `None` as default value for `last_update_sync`?
Now I change to use `Date.epoch` for its default value, does this resolve this issue?